### ISSUE TITLE
Zooming the current buffer closes the file in the buffer it was zoomed from.

### ISSUE THREAD

**Issue by tthkbw** [nyngwang/NeoZoom.lua#70]
The video on the github page shows the capability of zooming the current buffer from a p ane, and when the zoomed window is closed, you go back to the current buffer in the same pane.

I can't figure out how to set this up. I just use the defaults, and when I zoom a buffer, the buffer is closed in the pane I zoomed from. So, I zoom, do some editing, then close the zoom (:q) and now the pane I came from is empty.

How do I fix this?

**Comment by tthkbw** [comment]
Okay, so I figured this out, and it makes sense, but wasn't intuitive to me.

One must exit the zoomed floating window by using :NeoZoomToggle, and then the original buffer is still in place. Makes, sense, since I was using :q to exit the toggle and that closes the buffer.

**Comment by nyngwang** [comment]
> One must exit the zoomed floating window by using `:NeoZoomToggle`, and then the original buffer is still in place.

@tthkbw Exactly. I should probably update my README on this part since it could be not that apparent. I'm happy that you found the way out. (Feel free to ask questions / report bugs / provide advices / submit feature requests in the future.)

**Comment by tthkbw** [comment]
This sounds great--so I updated. But :q still closes the buffer and returns me to an empty buffer. Is there some setup to be done to enable the `autocmd` that you added?

I use Lazy as a package manager. It tells me I have commit 879f8d7 installed (I upgraded). When I look at the code for NeoZoom it does have the detect_colon_q function defined. And, if I do `:autocmd` I see this:

NeoZoom.lua  WinClosed
    *         <Lua 356: ~/.local/share/nvim/lazy/NeoZoom.lua/lua/neo-zoom/utils/autocmd.lua:7>

Suggestions?

**Comment by tthkbw** [comment]
Hmmm . . . Weird. As I described, it didn't work even though Lazy showed the updated version. It is possible that I had not restarted nvim after doing the update in Lazy. 

I just now started nvim again, and it all works as expected. This is great and I don't have to try to teach my old brain any new tricks!

I really like how this works. I often have many panes open. Now I can easily zoom one of them to focus on it without messing up my pane layout. Thanks!
